server/src/main/java/org/opensearch/telemetry/tracing/AttributeNames.java

```diff
@@ -64,4 +64,24 @@ private AttributeNames() {
      * Action Name.
      */
     public static final String TRANSPORT_ACTION = "action";
+
+    /**
+     * Index Name
+     */
+    public static final String INDEX = "index";
+
+    /**
+     * Shard ID
+     */
+    public static final String SHARD_ID = "shard_id";
+
+    /**
+     * Number of request items in bulk request
+     */
+    public static final String NUM_REQUEST_ITEMS  ="num_request_items";
```

Comment:
nit: rename it to NUM_BULK_ITEMS: "num_bulk_items"

Reply:
Ack. Will update it.